Arcs/Arcs.cpp

```diff
@@ -913,10 +1746,13 @@ void runArcs(const std::vector<std::string>& filenames) {
         << "\n -l " << params.min_links
         << "\n -m " << params.min_mult << '-' << params.max_mult
         << "\n -r " << params.error_percent
-        << "\n -s " << params.seq_id
         << "\n -v " << params.verbose
         << "\n -z " << params.min_size
         << "\n --gap=" << params.gap
+        << "\n -s " << params.seq_id << " (arcs specific option)"
```

Comment:
Is it possible to make this printing more ARCS/ARKS specific? Don't even print out the ARKS specific options if you are running in ARCS mode.